requirements.txt

```diff
@@ -1,7 +1,7 @@
 Django==1.9.7
 raphendyr-django-essentials==1.3.0
 celery==3.1.23
-pyrabbit==1.1.0
+pyrabbit==1.1.0 
```

Comment:
No need to add new whitespace to the end of the lines. You must check if the celery package is really still used.

Reply:
Done!